### ISSUE TITLE
Is it possible to use Blade instead of .php files for Server-rendered blocks?

### ISSUE THREAD

**Issue by Demonicious** [HansSchouten/Laravel-Pagebuilder#130]
Since this package aims to bring PHPageBuilder to Laravel, I was wondering if there's an ability to use Laravel Blade templates for your server-rendered blocks instead of usual `.php` files. It's not a big deal of course, but I believe it could probably improve the experience.

**Comment by HansSchouten** [comment]
I agree, that would be useful and I will keep it in mind, but for now I haven't had the time to integrate this yet.

**Comment by Demonicious** [comment]
In-any case, I really love this package. I may be able to help you with adding Laravel Blade support.

Look out for any PRs!

**Comment by humzakhaled** [comment]
can we integrate chartjs in blocks?
